src/libs/actions/BankAccounts.js

```diff
@@ -816,6 +816,27 @@ function updateReimbursementAccountDraft(bankAccountData) {
     Onyx.merge(ONYXKEYS.REIMBURSEMENT_ACCOUNT_DRAFT, bankAccountData);
 }
 
+/**
+ * Checks the given number is a valid US Routing Number
+ * @param {String} number
+ * @returns {Boolean}
+ */
+function validateRoutingNumber(number) {
+    let n = 0;
+    for (let i = 0; i < number.length; i += 3) {
+        n += (parseInt(number.charAt(i), 10) * 3)
+        + (parseInt(number.charAt(i + 1), 10) * 7)
+        + parseInt(number.charAt(i + 2), 10);
```

Comment:
I think it's nicer to indent these lines, it looks we do this most places in the app too
```suggestion
            + (parseInt(number.charAt(i + 1), 10) * 7)
            + parseInt(number.charAt(i + 2), 10);
```